bumpversion/__init__.py

```diff
@@ -317,13 +318,18 @@ class MercurialDoesNotSupportSignedTagsException(Exception):
     def __init__(self, message):
         self.message = message
 
+class UnkownPart(Exception):
+    def __init__(self, message):
+        self.message = message
+
 def keyvaluestring(d):
     return ", ".join("{}={}".format(k, v) for k, v in sorted(d.items()))
 
 class Version(object):
 
-    def __init__(self, values, original=None):
+    def __init__(self, values, config, original=None):
         self._values = dict(values)
+        self.config = config
```

Comment:
This is IMHO an internal property so should be named `self._config`

Reply:
Config is now the order and it has to be public because when comparing we first compare the orders cause if they are different the versions can not be comapred